### ISSUE TITLE
Question: Does adding more open issues to a board effect load time to a board?

### ISSUE THREAD

**Issue by benschac** [ZenHubIO/support#470]
Hi, 

I'm currently try to weigh my options of switching my teams work flow to a single milestone/board work flow.  

I was just curious if adding 300+ issue to one milestone would effect load time of a ZH board.

Thanks!

**Comment by azenMatt** [comment]
Hi @benschac, 

As we do need to load all issues that we show via the GitHub API (which sends paged responses) and then render them on the board. The more issues that you have, the longer this process will take. 

At 300, you are not going to see any problems, for an idea of the time - you can load the board on this repo which is roughly 250 issues (200 open + 50 closed) 

As you approach the 1000+ issue mark, it starts to get more noticeable - but we still have users with over 50 boards merged together that load 1000's of issues! 

One other quick point i'l make is that board performance is a big focus for us right now - we have released a several updates in the last month that improve performance and have several more lined up - so that the number of issues you want to load will have a smaller and smaller impact on performance. 

In summary - in your position, i'd work from a single board :smile: 

Please let me know how you get on!

**Comment by benschac** [comment]
will do, thanks for the speedy response @azenMatt :)
